sklearn/preprocessing/data.py

```diff
@@ -2901,6 +2902,95 @@ def power_transform(X, method='box-cox', standardize=True, copy=True):
     return pt.fit_transform(X)
 
 
+class Winsorizer(BaseEstimator, TransformerMixin):
+    """Transforms each feature by clipping from below at the pth quantile
```

Comment:
"Clips each feature at the pth quantile from above and below"

should be sufficient.

Add Read more in the User Guide (as in most other estimator docstrings) once a section is in the user guide.